internal/attachment/delivery/http/middleware/middleware.go

```diff
@@ -0,0 +1,21 @@
+package middleware
+
+import "github.com/labstack/echo/v4"
+
+// GoMiddleware represent the data-struct for middleware
+type GoMiddleware struct {
+	// another stuff , may be needed by middleware
+}
+
+// CORS will handle the CORS middleware
+func (m *GoMiddleware) CORS(next echo.HandlerFunc) echo.HandlerFunc {
+	return func(c echo.Context) error {
+		c.Response().Header().Set("Access-Control-Allow-Origin", "*")
```

Comment:
As far as I know, echo has its own way to work with CORS. It's no need to handle it ourselves.

But for now, I suggest you should not introduce CORS here. Please remove and submit other PRs.